agent/recognition/binarymatch.py

```
@@ -106,132 +136,144 @@ def analyze(
-           代码内部自动映射到 Pillow 标准 (H:0-255)。
+        HSV 形状匹配识别器（Pillow 无 OpenCV 依赖版，兼容 Windows ARM64）
+
+        工作流程：
+        1. 固定截图 argv.image，全程复用，不重复截图。
+        2. 按 HSV 阈值（支持多组 OR）生成白底黑形状掩膜图，交给目标节点识别。
+        3. 若 edge_assist=true，HSV 失败后追加边缘 AND 掩膜再试一次。
+
+        参数格式与完整示例见模块顶部注释块。
         """
         try:
+            # 0. 固定截图，全程复用
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)